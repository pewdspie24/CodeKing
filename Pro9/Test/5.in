10
2665741111 6632727215
2569668605 3671299218
4258746505 7598938702
1751073618 4126693007
1999758144 2290613932
4218408800 4279071331
2113999695 4353733347
4151742894 4467867357
4206139380 6337475823
455030183 1035389971
