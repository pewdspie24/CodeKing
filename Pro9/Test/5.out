35232365
11447405
24107603
24774338
4313357
754194
23930291
3840795
17161948
5274227
